www/js/diary/services.js

```diff
@@ -85,12 +85,12 @@ angular.module('emission.main.diary.services', ['emission.services'])
         'text-align': 'center',
         'background-color': '#ff5251',
         'color': 'black',
-        'width': '48px',
+        'width': '18%',
         'height': '16px',
```

Comment:
I don't think it's necessary to make this taller in larger screen because it just show the time, and as long as the time is clear to see, it should be fine. What do u think?